baseplate/lib/metrics.py

```diff
@@ -299,16 +311,24 @@ def send(self, elapsed: float, sample_rate: float = 1.0) -> None:
 
         This can be useful when the timing was managed elsewhere and we just
         want to report the result.
-
         :param elapsed: The elapsed time in seconds to report.
-
         """
-        serialized = self.name + (f":{(elapsed * 1000.0):g}|ms".encode())
+
+        formatted_tags = _format_tags(self.tags)
+        if formatted_tags:
+            parts = [self.name + formatted_tags.encode() + f":{(elapsed * 1000.0):g}|ms".encode()]
+        else:
+            parts = [self.name + f":{(elapsed * 1000.0):g}|ms".encode()]
```

Comment:
```suggestion
        serialized = self.name
        if self.tags:
            serialized += _format_tags(self.tags)
        serialized += f":{(elapsed * 1000.0):g}|ms".encode()
```

would be a bit less repetitive. though this probably bears a bit of benchmarking?